### DISCUSSION TITLE
warning "@automock/jest > jest-mock-extended@2.0.9" has incorrect peer dependency

### DISCUSSION THREAD

**Discussion by jasonmacdonald** [automock/automock#102] [General]
Hi, this is likely a minor thing, but any chance we can update the `jest-mock-extended`  to version`3` as it currently complains during `yarn install` about ```warning "@automock/jest > jest-mock-extended@2.0.9" has incorrect peer dependency "typescript@^3.0.0 || ^4.0.0".``` when you are using `Typescript v5`. 

The package line [can be found here ](https://github.com/automock/automock/blob/0576ae5503a6f77e90f0d0bdde427c5493ccc318/packages/testbeds/jest/package.json#L39)

I'm not sure if there's a specific reason the older version is needed or not. I'm happy to submit a PR if you are OK with it.

Cheers, and thanks for the great package!

**Comment by omermorad** [comment]
Hey @jasonmacdonald,

In preparation for Automock 2.0, we have decided to deprecate the 'jest-mock-extended' dependency. We are instead implementing in-house, brand-new features. In the near future, I'll post an update 🚀

**Reply by omermorad** [reply]
@jasonmacdonald fixed in version 2.0 :)
